Misc/NEWS.d/next/Documentation/2020-09-22-12-32-16.bpo-41825.npcaCb.rst

```diff
@@ -0,0 +1,3 @@
+Restructured the documentation for the ``os.wait*()`` family of functions,
```

Comment:
```suggestion
Restructured the documentation for the :func:`os.wait* <os.wait>` family of functions,
```

Link the name to the baseline function in the category, so readers can jump to the appropriate docs more easily.